internal/cli/deployments/connect_test.go

```diff
@@ -192,6 +147,29 @@ func TestRun_ConnectAtlas(t *testing.T) {
 `, buf.String())
 }
 
+func TestRun_PostRun(t *testing.T) {
```

Comment:
```suggestion
func TestPostRun(t *testing.T) {
```
unless you change the method to have the opts to be a receiver in which case `TestConnectOpts_PostRun`

Reply:
updated to TestPostRun for now